app/src/main/java/net/gsantner/Constants.java

```diff
@@ -0,0 +1,5 @@
+package net.gsantner;
+
+public interface Constants {
+    String ENCRYPTION_EXTENSION = ".jenc";
```

Comment:
Can be moved to the JavaPasswordbasedCryption.java class

Reply:
Done